From fae7964790eeba2032d0fd96d79beec6505657da Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Mon, 17 Sep 2018 09:24:40 +0800
Subject: [PATCH] ARM: rockchip_defconfig: Enable CONFIG_COREDUMP

vts-testcase/kernel/api/proc/KernelProcFileApiTest
had added tests for the core_pattern.

https://android.googlesource.com/kernel/configs
5ed73dc4d482 ("add missing configs")

CONFIG_COREDUMP and CONFIG_ELF_CORE are recommended for diagnostic
purposes.

# CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
came from android-recommended.config

Change-Id: Ib16fcc6c126e8415976fea2d7e193565866bcf2d
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index a53624f30cdf..a5c7dbbaa3dc 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -74,7 +74,7 @@ CONFIG_CPU_IDLE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_KERNEL_MODE_NEON=y
-# CONFIG_COREDUMP is not set
+# CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
-- 
2.35.3

